cmd/atlas/internal/cmdapi/migrate_test.go

```diff
@@ -582,3 +588,12 @@ func countFiles(t *testing.T, p string) int {
 	require.NoError(t, err)
 	return len(files)
 }
+
+func sed(t *testing.T, r, p string) {
+	args := []string{"-i"}
+	if runtime.GOOS == "darwin" {
+		args = append(args, ".bk")
+	}
```

Comment:
It fails locally on darwin.

Reply:
Darwin has no GNU sed. You need to install it.